app/export/courses.rb

```diff
@@ -0,0 +1,31 @@
+# Takes an existing semester live on umd.io and exports it to JSON
+# TODO: Update to use v1 schema, but only when courses_importer does too
+# TODO: Use logger
+
+require 'open-uri'
+require 'json'
+
+def export_sem(sem)
+  courses = []
+  n = 0
+
+  while (res = open("https://api.umd.io/v0/courses?semester=#{sem}&expand=sections&page=#{n}").read)
```

Comment:
actually should probably make this configurable easier, e.g. to export from local instance